code/modules/projectiles/guns/ballistic.dm

```diff
@@ -107,17 +107,17 @@
 
 
 /obj/item/gun/ballistic/process_chamber(empty_chamber = TRUE, from_firing = TRUE, chamber_next_round = TRUE)
-	if(!semi_auto && from_firing)
+	if (!semi_auto && from_firing)
```

Comment:
leave this as was, please. iirc we had a precedent set where you shouldn't insert/remove spaces if you're not actually changing anything.

Reply:
I'll fix this when I get home